packages/babel-parser/src/plugins/flow/index.js

```diff
@@ -2117,7 +2117,8 @@ export default (superClass: Class<Parser>): Class<Parser> =>
       if (member.declare) {
         if (
           member.type !== "ClassProperty" &&
-          member.type !== "ClassPrivateProperty"
+          member.type !== "ClassPrivateProperty" &&
+          member.type !== "PropertyDefinition" // estree
```

Comment:
Maybe the following for more context?

```suggestion
          member.type !== "PropertyDefinition" // Used by estree plugin
```